chapter-06/integration-tests/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "integration-tests"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
```

Comment:
I don't think we need all of these deps here. The ones we need (probably except `serde`) can probably be moved to `dev-dependencies` since we do everything inside a `tests` package, right?